corehq/apps/domain/forms.py

```diff
@@ -804,7 +806,7 @@ def __init__(self, can_edit_eula, *args, **kwargs):
                 StrictButton(
```

Comment:
Can you use `FormActions` to surround `StrictButton`? It will take care of the offset, so you won't need `col-lg-offset-2` below. Thanks! https://staging.commcarehq.org/styleguide/forms/#b3-migration talks about this at the end of the checklist for Existing Crispy Forms.